### ISSUE TITLE
docker deployment error

### ISSUE THREAD

**Issue by lainbo** [Songkeys/Translateer#20]
When I use `git clone https://github.com/Songkeys/translateer.git` and `docker-compose up -d` on the server, an error occurred. Here is the log
```
> translateer@2.0.0 bootstrap
> npm ci && npm run build && npm run prod


added 244 packages, and audited 245 packages in 22s

17 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

> translateer@2.0.0 build
> rm -rf dist && tsc && cp -R src/public dist/public

npm ERR! Missing script: "prod"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2024-01-04T12_16_32_229Z-debug-0.log
npm notice 
npm notice New patch version of npm available! 10.2.4 -> 10.2.5
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.2.5>
npm notice Run `npm install -g npm@10.2.5` to update!
npm notice 
```
I use Debian12 vps
When I submit the rollback to `git checkout 21e6cd64fcdfdc10a5bf56d01b6d3b0062acb347`, the program works fine.

**Comment by lainbo** [comment]
When I submit the rollback to `git checkout 21e6cd64fcdfdc10a5bf56d01b6d3b0062acb347`, the program works fine.

**Comment by songkeys** [comment]
Sorry about this. I've fixed it in commit <https://github.com/Songkeys/Translateer/commit/04a610cc11281bdc67ce5e12aa90e073cdea4d32>. Could you try again?

**Comment by lainbo** [comment]
> Sorry about this. I've fixed it in commit [04a610c](https://github.com/Songkeys/Translateer/commit/04a610cc11281bdc67ce5e12aa90e073cdea4d32). Could you try again?

thank you for fixing it, it's working properly now
